include/SubPanel/tpls/SubPanelTiles.tpl

```diff
@@ -73,6 +73,14 @@
 {/if}
 <script>
     var ModuleSubPanels = {$module_sub_panels};
+    {literal}
+    setTimeout(function() {
+        // Filter subpanels to show the current tab
+        if(typeof SUGAR.subpanelUtils.currentSubpanelGroup !== "undefined") {
+            SUGAR.subpanelUtils.loadSubpanelGroup(SUGAR.subpanelUtils.currentSubpanelGroup);
+        }
+    }, 500);
```

Comment:
Issue with a slow loading screen - should be interval until successful in case of missing it.